tests/jsx_test.ts

```diff
@@ -0,0 +1,107 @@
+import {
```

Comment:
You can use the snapshot tests instead so that you don't have to setup output fixtures nor repeat the logic that calls `transpile`. It will also cover `bundle`.

Reply:
Bundle doesn't seem to work with `react-jsx` and `react-jsxdev`. 
It throws an error (`Error: Unable to output during bundling`). 

This can be avoided by explicitly declaring the import source with `/** @jsxImportSource https://esm.sh/react@17.0.2 */` in the input file.

Apparently, this issue has already been fixed for `deno bundle` (https://github.com/denoland/deno/issues/13389).
Does this have to be implemented again for `deno_emit`?